### PR TITLE
Add swipe refresh element

### PR DESCRIPTION
**Pull request by dmitriigriazin**

### CODE REVIEW

File: test-app/src/androidTest/java/com/avito/android/ui/test/SwipeRefreshTest.kt [hunk 1]
```diff
@@ -12,11 +12,32 @@ class SwipeRefreshTest {
     val rule = screenRule<SwipeRefreshActivity>()
 
     @Test
-    fun recyclerView_refresh_onPullToRefresh() {
```
Comment (dmitriigriazin) [inline test-app/src/androidTest/java/com/avito/android/ui/test/SwipeRefreshTest.kt:15]: This one was lying, there is no recyclerView in layout

File: test-app/src/androidTest/java/com/avito/android/ui/test/SwipeRefreshTest.kt [hunk 2]
```diff
@@ -12,11 +12,32 @@ class SwipeRefreshTest {
...
+    fun refresh_onPullToRefresh_listAction() {
         rule.launchActivity(null)
 
         Screen.swipeRefresh.list.actions.pullToRefresh()
```
Comment (dmitriigriazin) [inline test-app/src/androidTest/java/com/avito/android/ui/test/SwipeRefreshTest.kt:18]: To be honest, I don't think that `ListElement` is a right place for the `pullToRefresh` action. I do not touch it in order to cause no conflicts

**Comment by dsvoronin** [comment]
After #6, Push something to re-trigger checks please

**Review by dsvoronin — CHANGES_REQUESTED** [review]
Need to update README

File: README.md [hunk 3]
```diff
@@ -2,7 +2,29 @@
...
 
-Not now. Work in progress
+Not now. Work in progress.
+
+######Roadmap: TBD
+
+One does not simply say WIP. 
+We should announce what the DoD is. 
+
+## Features
```
Comment (dsvoronin) [inline README.md]: it's better to have `Features` block under configuration, because it tends to be huge

File: README.md [hunk 4]
```diff
@@ -2,7 +2,29 @@
 
 ## Should I use it
 
-Not now. Work in progress
+Not now. Work in progress.
+
+######Roadmap: TBD
+
+One does not simply say WIP. 
+We should announce what the DoD is.
```
Comment (dsvoronin) [inline README.md]: Let's say:
1. we have full test coverage of current features
2. we have docs about all features

that's 1.0, than we rethink some of weak points and address them in 2.0

File: README.md [hunk 5]
```diff
@@ -2,7 +2,29 @@
...
+## Features
+
+Handy checks and actions for common Android UI elements! 
+
+`TBD: Simple samples? ` 
+
+```
+view.click()
+
+swipeRefreshElement.checks.isRefreshing()
```
Comment (dsvoronin) [inline README.md:37]: We need two things here: 
1. element instantiation
2. what element offers: checks, actions as call examples

**Review by dsvoronin — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (dmitriigriazin) [reply README.md]: Need to make a checklist. Will put an issue on you.

**Comment by dmitriigriazin** [comment]
@dsvoronin let's merge it, leaving Readme work after #11

**Review by dsvoronin — APPROVED** [review]

**Pull request merged**